Specs/ASTMapSpec.m

```diff
@@ -98,6 +102,24 @@ - (void)testEverything {
                                           }));
         }];
 
+        [XCTContext runActivityNamed:@"should return a dictioanry" block:^(id<XCTActivity> _Nonnull activity){
```

Comment:
Can you move this out of the `for arrays` scope? Also typo in `dictioanry`.

Reply:
Actually the comment should be an array, and the target is a dictionary, so not sure if for arrays or for dictionaries is the right place here. You sure about the for arrays scope?